sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -610,3 +610,17 @@ def _create_combine_phase_operation(
           factory.state_sampler),
       transform_proto.unique_name,
       consumers)
+
+
+@BeamTransformFactory.register_urn(urns.FLATTEN_TRANSFORM, None)
```

Comment:
Did "urns.FLATTEN_TRANSFORM" need to be updated to match Java?

Reply:
Yes, right now it differs from the one being used in Java. Since PR #4310 is being worked on to solve the discrepancies, we can wait to merge this PR until after that one is merged, or I can just make a new PR later to fix the URN. What do you think would be better? I'm leaning towards the second option.

Reply:
OK, let's get this one in and hopefully #4310 will be merged soon.